app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -71,53 +72,56 @@ fun ViewProfileScreen(
             tabList = LIST_TOP_LEVEL_DESTINATION,
             selectedItem = Route.PROFILE)
       }) { innerPadding ->
-        Column(
-            modifier =
-                Modifier.fillMaxSize()
-                    .padding(innerPadding)
-                    .padding(16.dp)
-                    .testTag("ProfileContent"),
-            horizontalAlignment = Alignment.CenterHorizontally) {
-              if (isEditing) {
-                EditProfileContent(
-                    firstName = firstName,
-                    lastName = lastName,
-                    username = username,
-                    profilePictureUrl = profilePictureUrl,
-                    onFirstNameChange = { firstName = it },
-                    onLastNameChange = { lastName = it },
-                    onUsernameChange = { username = it },
-                    onImageClick = { imagePickerLauncher.launch("image/*") },
-                    onSave = {
-                      userViewModel.updateUser(
-                          userState?.copy(
-                              firstName = firstName,
-                              lastName = lastName,
-                              username = username,
-                              profilePictureUrl = profilePictureUrl) ?: return@EditProfileContent)
-                      userViewModel.getUserById(userState?.userId ?: "")
-                      isEditing = false
-                    },
-                    onCancel = {
-                      firstName = userState?.firstName ?: ""
-                      lastName = userState?.lastName ?: ""
-                      username = userState?.username ?: ""
-                      profilePictureUrl = userState?.profilePictureUrl ?: ""
-                      isEditing = false
-                    })
-              } else {
-                DisplayProfileContent(
-                    firstName = firstName,
-                    lastName = lastName,
-                    username = username,
-                    profilePictureUrl = profilePictureUrl,
-                    onEditClick = { isEditing = true })
-
-                Spacer(modifier = Modifier.height(24.dp))
-
-                FavoriteParkingsSection(userViewModel)
+        Box(Modifier.fillMaxSize().padding(innerPadding)) {
+          authenticator.SignOutButton {
+            userViewModel.setCurrentUser(null)
+            navigationActions.navigateTo(Route.AUTH)
```

Comment:
The solution for this is to call `navigateTo(TopLevelDestination)` instead of `navigateTo(String)`, because the one with `TopLevelDestination` pop the stack of the navigation actions.

Reply:
Yes, that's a good idea, thanks for the solution @sean-prz I think it should fit well in your PR on navigation bugfix!